app/src/js/components/ReconciliationReports/report-heading.js

```diff
@@ -34,11 +35,11 @@ const ReportHeading = ({
   ];
 
   const formattedStartTime = startTime
-    ? new Date(startTime).toLocaleDateString()
+    ? moment(startTime).utc().toLocaleString()
     : 'missing';
 
   const formattedEndTime = endTime
-    ? new Date(endTime).toLocaleDateString()
+    ? moment(endTime).utc().toLocaleString()
```

Comment:
Does `toLocaleString()` still do anything with `moment.js`?

Also, this date formatting doesn't match the mockups, but not sure how important that is.

<img width="555" alt="Screen Shot 2020-08-24 at 1 07 54 PM" src="https://user-images.githubusercontent.com/16179480/91075242-c0afd900-e60b-11ea-85f8-4b6d80f55185.png">

[Mockup](https://xd.adobe.com/view/8a2afd0f-2e25-4814-79a3-11f3cc85c0c3-fc1b/screen/433a3107-824c-4120-a6bb-ce790791ca9d/)

Reply:
Yeah, we can ask @jjmccoy, But since we have timestamps going into the reports, we need to have timestamps coming out.  
The toLocaleString() does appear to format 
```
 moment().utc().toLocaleString()
'Mon Aug 24 2020 17:20:33 GMT+0000'
```
But I didn't look at the mockup.  I can see now wanting the Day of Week and maybe not the TZ offset?

Reply:
Ah yeah, you're right it does. Though it seems like using `moment(startTime).utc()` without still works/displays a string. But probably best to have it.

Agreed re: the formatting and needing the timestamp. Definitely think we can nix day of the week though. And the TZ offset, as long as we're using UTC everywhere else (which I think we are?)

Reply:
`(which I think we are?)` we should be....

I think moment(datething).utc() will create a utc moment object that is coerced into a string when we print it and we should probably be explicit.  I will find a better format string and let you and Juanisa decide if it's good enough.

Reply:
Hmm...

Reply:
Can we separate that please: The 'Date Range' and then under the 'Timestamp' and they will both be stacked in the same row with the report title? Please keep in mind they can select 'created', 'ingested', 'reingested', 'granule start time', or 'granule end time' as their timestamp as well.

Reply:
I'm updating this with the format that got the most votes, we can address new inputs in an architecture meeting.